### ISSUE TITLE
Hive doesn't follow the normal login sequence

### ISSUE THREAD

**Issue by HashimTheArab** [Sandertv/gophertunnel#214]
The conn expects a ServerToClientHandshake, and a PlayStatus
https://github.com/Sandertv/gophertunnel/blob/2b48a19c84f515175e8447795e4613a520494b91/minecraft/dial.go#L236

Hive does not send the play status, but immediately sends StartGame, resulting in a closed connection

## Cubecraft
Handling packet, *packet.ServerToClientHandshake
Handling packet, *packet.PlayStatus
Handling packet, *packet.ResourcePacksInfo
Handling packet, *packet.ResourcePackStack
Handling packet, *packet.StartGame

## Hive
Handling packet, *packet.ServerToClientHandshake
conn.receive: Handling packet id 11 (packet.StartGame)
[log.fatal]: [Proxy]: could not join: dial minecraft 10.0.0.92:49308->148.113.162.177:19132: use of closed network connection

**Comment by ismaileke** [comment]
Yes, I noticed this too. Why doesn't The Hive follow the normal login sequence?

**Comment by ismaileke** [comment]
> Yes, I noticed this too. Why doesn't The Hive follow the normal login sequence?

To solve this problem it is necessary to emulate the behavior of the Minecraft Client
